example/tk/readme.md

```diff
@@ -8,16 +8,21 @@ better at demonstrating trace discovery flows using Loki and other tools.
 If you're convinced this is the place for you then keep reading!
 
 ### Initial Steps
-
 The Jsonnet is meant to be applied to with [tanka](https://github.com/grafana/tanka).  To test the jsonnet locally requires:
 
 - k3d > v3.2.0
 - tanka > v0.12.0
 
-Create a cluster
+Create a cluster with 3 nodes
+
+```console
+k3d cluster create tempo --servers 3 --api-port 6443 --port "16686:80@loadbalancer"
```

Comment:
I had to add `--servers 3` otherwise it was not possible to query Tempo in the microservices config.

Reason:
- due to anti-affinity rules, it's not possible to run 3 ingesters on a single node
- if there are not enough ingesters online queries fail in the query-frontend with
  ```
  error finding ingesters in Querier.FindTraceByID: too many unhealthy instances in the ring
  ```

This is not an issue in the helm example... so we maybe we can avoid this by changing some jsonnet config instead 🤔

Reply:
I think that the >=3 servers requirement is reasonable! But if we can do some Jsonnet magic to support these <3 node deployments, it could be interesting, agree.

Reply:
Can we run with replication factor 1? Does it still fail?

Reply:
Oh! I can try to override the replication config, I doesn't work if I just override `replicas`.

Reply:
the "jsonnet magic" path to just remove the anti affinity rules is the most attractive to me.  

however, setting "--servers=3" isn't that big of a deal so if this is a lot of work please feel free to just leave this. since this is meant to be a semi production setup i'd like to keep RF=3 and have 3 ingesters.

Reply:
I have been able to remove the affinity settings using this jsonnet: https://github.com/kvrhdn/tempo/blob/707-update-docs/example/tk/tempo-microservices/main.jsonnet#L71-L80